package.json

```diff
@@ -105,21 +100,18 @@
     "ember-router-generator": "^1.0.0",
     "escape-string-regexp": "^1.0.3",
     "exists-sync": "0.0.3",
-    "exit": "^0.1.2",
```

Comment:
This is used here: https://github.com/ember-cli/ember-cli/blob/058528977255d0e366b8a1760668875cfadf9067/bin/ember#L8